.github/workflows/smoke-test-release.yml

```diff
@@ -1,185 +1,690 @@
 name: Smoke test release
 on:
+    release:
+        types: [published]
     workflow_dispatch:
         inputs:
-            release_id:
-                description: 'WooCommerce Release Id'
+            tag:
+                description: 'WooCommerce Release Tag'
```

Comment:
Use release tag instead of release ID, because release tag is easier to obtain (it's just the same as the release version like `6.9.3` or `7.0.0-rc.1`)